README.md

```diff
@@ -58,10 +58,46 @@ build/$ ctest . --output-on-failure
 
 The bci.config file is used by our internal validation environment, please just ignore it.
 
+### Build with dilithium support
+
+Dilithium is an **optional** feature provided by MoCOCrW.
+
+This feature depends on [reference implementation of the Dilithium signature scheme](https://github.com/pq-crystals/dilithium/)
+since OpenSSL still doesn't have a support for Dilithium. In order to sucessfully compile MoCOCrW
+with Dilithium feature the following adaptations are necessary. 
```

Comment:
```suggestion
since OpenSSL still doesn't have a support for Dilithium. The following adaptations are necessary
in order to successfully compile MoCOCrW with the Dilithium feature . 
```